qna/freeform/001.md

```diff
@@ -0,0 +1,29 @@
+## Freeform question
+A “Software Bill of Materials” (SBOM) is
+effectively a nested inventory,
+a list of ingredients that make up
+software components.
+
+The recent Executive Order on Supply Chain Cybersecurity
+defined SBOM as a formal record containing the details and supply chain
+relationships of various components used in building software.
+Software developers and vendors often create products by
+assembling existing open source and commercial software components.
+The SBOM enumerates these components in a product.
+It is analogous to a list of ingredients on food packaging.
+An SBOM is useful to those who develop or manufacture software,
+those who select or purchase software, and those who operate software.
+
+
+# Question:
+What does SBOM stand for?
+
+## Answers
+* 001
```

Comment:
@sigu what happens in a case where we have more than one answer being correct?

Reply:
Good call,, we can handle that if needed